[2025-02-22T23:43:16.655-0500] {processor.py:186} INFO - Started process (PID=26520) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:43:16.656-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-22T23:43:16.659-0500] {logging_mixin.py:190} INFO - [2025-02-22T23:43:16.658-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:43:16.665-0500] {logging_mixin.py:190} INFO - [2025-02-22T23:43:16.664-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-22T23:43:16.665-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:43:16.708-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.062 seconds
[2025-02-22T23:44:26.067-0500] {processor.py:186} INFO - Started process (PID=27025) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:44:26.069-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-22T23:44:26.072-0500] {logging_mixin.py:190} INFO - [2025-02-22T23:44:26.071-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:44:26.079-0500] {logging_mixin.py:190} INFO - [2025-02-22T23:44:26.078-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-22T23:44:26.079-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:44:26.112-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.054 seconds
[2025-02-22T23:49:00.836-0500] {processor.py:186} INFO - Started process (PID=27647) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:49:00.838-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-22T23:49:00.841-0500] {logging_mixin.py:190} INFO - [2025-02-22T23:49:00.840-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:49:00.850-0500] {logging_mixin.py:190} INFO - [2025-02-22T23:49:00.848-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-22T23:49:00.850-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:49:00.883-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.057 seconds
[2025-02-22T23:50:03.872-0500] {processor.py:186} INFO - Started process (PID=28102) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:50:03.874-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-22T23:50:03.878-0500] {logging_mixin.py:190} INFO - [2025-02-22T23:50:03.877-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:50:03.886-0500] {logging_mixin.py:190} INFO - [2025-02-22T23:50:03.885-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-22T23:50:03.887-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:50:03.919-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.057 seconds
[2025-02-22T23:51:01.147-0500] {processor.py:186} INFO - Started process (PID=28497) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:51:01.149-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-22T23:51:01.152-0500] {logging_mixin.py:190} INFO - [2025-02-22T23:51:01.151-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:51:01.159-0500] {logging_mixin.py:190} INFO - [2025-02-22T23:51:01.157-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-22T23:51:01.159-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:51:01.194-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.057 seconds
[2025-02-22T23:51:58.754-0500] {processor.py:186} INFO - Started process (PID=28924) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:51:58.756-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-22T23:51:58.759-0500] {logging_mixin.py:190} INFO - [2025-02-22T23:51:58.758-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:51:58.768-0500] {logging_mixin.py:190} INFO - [2025-02-22T23:51:58.766-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-22T23:51:58.768-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:51:58.800-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.055 seconds
[2025-02-22T23:53:00.870-0500] {processor.py:186} INFO - Started process (PID=29371) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:53:00.871-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-22T23:53:00.874-0500] {logging_mixin.py:190} INFO - [2025-02-22T23:53:00.873-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:53:00.881-0500] {logging_mixin.py:190} INFO - [2025-02-22T23:53:00.880-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-22T23:53:00.881-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:53:00.911-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.050 seconds
[2025-02-22T23:53:34.308-0500] {processor.py:186} INFO - Started process (PID=29595) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:53:34.310-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-22T23:53:34.314-0500] {logging_mixin.py:190} INFO - [2025-02-22T23:53:34.314-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:53:34.326-0500] {logging_mixin.py:190} INFO - [2025-02-22T23:53:34.325-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-22T23:53:34.326-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:53:34.358-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.060 seconds
[2025-02-22T23:54:39.384-0500] {processor.py:186} INFO - Started process (PID=30067) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:54:39.389-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-22T23:54:39.394-0500] {logging_mixin.py:190} INFO - [2025-02-22T23:54:39.393-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:54:39.412-0500] {logging_mixin.py:190} INFO - [2025-02-22T23:54:39.403-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-22T23:54:39.413-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:54:39.470-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.104 seconds
[2025-02-22T23:57:50.144-0500] {processor.py:186} INFO - Started process (PID=30530) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:57:50.147-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-22T23:57:50.150-0500] {logging_mixin.py:190} INFO - [2025-02-22T23:57:50.149-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:57:50.159-0500] {logging_mixin.py:190} INFO - [2025-02-22T23:57:50.158-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-22T23:57:50.160-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:57:50.194-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.060 seconds
[2025-02-22T23:58:37.532-0500] {processor.py:186} INFO - Started process (PID=30889) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:58:37.534-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-22T23:58:37.537-0500] {logging_mixin.py:190} INFO - [2025-02-22T23:58:37.536-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:58:37.546-0500] {logging_mixin.py:190} INFO - [2025-02-22T23:58:37.545-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-22T23:58:37.546-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:58:37.583-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.062 seconds
[2025-02-22T23:59:45.360-0500] {processor.py:186} INFO - Started process (PID=31315) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:59:45.412-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-22T23:59:45.417-0500] {logging_mixin.py:190} INFO - [2025-02-22T23:59:45.416-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:59:45.460-0500] {logging_mixin.py:190} INFO - [2025-02-22T23:59:45.459-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-22T23:59:45.461-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-22T23:59:45.844-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.528 seconds
[2025-02-23T00:00:26.455-0500] {processor.py:186} INFO - Started process (PID=31585) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:00:26.457-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-23T00:00:26.460-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:00:26.460-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:00:26.468-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:00:26.467-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-23T00:00:26.468-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:00:26.498-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.052 seconds
[2025-02-23T00:01:06.179-0500] {processor.py:186} INFO - Started process (PID=31862) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:01:06.181-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-23T00:01:06.184-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:01:06.183-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:01:06.197-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:01:06.195-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-23T00:01:06.198-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:01:06.237-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.074 seconds
[2025-02-23T00:02:05.038-0500] {processor.py:186} INFO - Started process (PID=32304) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:02:05.042-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-23T00:02:05.045-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:02:05.044-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:02:05.063-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:02:05.061-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-23T00:02:05.064-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:02:05.137-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.112 seconds
[2025-02-23T00:11:01.098-0500] {processor.py:186} INFO - Started process (PID=34398) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:11:01.105-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-23T00:11:01.108-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:11:01.107-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:11:01.117-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:11:01.116-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-23T00:11:01.118-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:11:01.148-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.058 seconds
[2025-02-23T00:12:01.101-0500] {processor.py:186} INFO - Started process (PID=34490) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:12:01.102-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-23T00:12:01.106-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:12:01.105-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:12:01.112-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:12:01.111-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-23T00:12:01.112-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:12:01.141-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.054 seconds
[2025-02-23T00:13:17.036-0500] {processor.py:186} INFO - Started process (PID=34603) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:13:17.038-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-23T00:13:17.042-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:13:17.041-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:13:17.048-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:13:17.047-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-23T00:13:17.049-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:13:17.084-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.057 seconds
[2025-02-23T00:14:13.397-0500] {processor.py:186} INFO - Started process (PID=34676) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:14:13.399-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-23T00:14:13.402-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:14:13.402-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:14:13.408-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:14:13.407-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-23T00:14:13.409-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:14:13.437-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.048 seconds
[2025-02-23T00:15:23.870-0500] {processor.py:186} INFO - Started process (PID=34759) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:15:23.873-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-23T00:15:23.875-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:15:23.875-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:15:23.886-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:15:23.885-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-23T00:15:23.886-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:15:23.924-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.063 seconds
[2025-02-23T00:16:22.385-0500] {processor.py:186} INFO - Started process (PID=34874) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:16:22.387-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-23T00:16:22.390-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:16:22.389-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:16:22.398-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:16:22.397-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-23T00:16:22.398-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:16:22.430-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.055 seconds
[2025-02-23T00:17:27.669-0500] {processor.py:186} INFO - Started process (PID=34945) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:17:27.676-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-23T00:17:27.684-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:17:27.683-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:17:27.696-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:17:27.692-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-23T00:17:27.697-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:17:27.759-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.109 seconds
[2025-02-23T00:18:27.697-0500] {processor.py:186} INFO - Started process (PID=35019) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:18:27.699-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-23T00:18:27.702-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:18:27.702-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:18:27.708-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:18:27.707-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-23T00:18:27.709-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:18:27.744-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.055 seconds
[2025-02-23T00:21:43.608-0500] {processor.py:186} INFO - Started process (PID=35158) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:21:43.614-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-23T00:21:43.617-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:21:43.616-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:21:43.627-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:21:43.626-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-23T00:21:43.628-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:21:43.654-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.055 seconds
[2025-02-23T00:22:44.093-0500] {processor.py:186} INFO - Started process (PID=35226) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:22:44.099-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-23T00:22:44.104-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:22:44.102-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:22:44.116-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:22:44.115-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-23T00:22:44.117-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:22:44.164-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.087 seconds
[2025-02-23T00:23:45.130-0500] {processor.py:186} INFO - Started process (PID=35291) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:23:45.132-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-23T00:23:45.136-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:23:45.136-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:23:45.143-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:23:45.142-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-23T00:23:45.144-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:23:45.177-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.055 seconds
[2025-02-23T00:24:48.601-0500] {processor.py:186} INFO - Started process (PID=35369) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:24:48.603-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-23T00:24:48.606-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:24:48.605-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:24:48.612-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:24:48.611-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-23T00:24:48.613-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:24:48.643-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.049 seconds
[2025-02-23T00:25:48.035-0500] {processor.py:186} INFO - Started process (PID=35436) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:25:48.042-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-23T00:25:48.044-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:25:48.044-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:25:48.050-0500] {logging_mixin.py:190} INFO - [2025-02-23T00:25:48.049-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-23T00:25:48.051-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T00:25:48.082-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.055 seconds
[2025-02-23T02:31:28.788-0500] {processor.py:186} INFO - Started process (PID=75931) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T02:31:28.790-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-23T02:31:28.797-0500] {logging_mixin.py:190} INFO - [2025-02-23T02:31:28.795-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T02:31:28.804-0500] {logging_mixin.py:190} INFO - [2025-02-23T02:31:28.803-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-23T02:31:28.805-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T02:31:28.841-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.064 seconds
[2025-02-23T02:32:28.849-0500] {processor.py:186} INFO - Started process (PID=76367) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T02:32:28.856-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-23T02:32:28.859-0500] {logging_mixin.py:190} INFO - [2025-02-23T02:32:28.858-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T02:32:28.865-0500] {logging_mixin.py:190} INFO - [2025-02-23T02:32:28.864-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-23T02:32:28.866-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T02:32:28.896-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.056 seconds
[2025-02-23T02:33:46.204-0500] {processor.py:186} INFO - Started process (PID=76765) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T02:33:46.206-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-23T02:33:46.210-0500] {logging_mixin.py:190} INFO - [2025-02-23T02:33:46.209-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T02:33:46.218-0500] {logging_mixin.py:190} INFO - [2025-02-23T02:33:46.217-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-23T02:33:46.219-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T02:33:46.255-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.063 seconds
[2025-02-23T02:34:30.000-0500] {processor.py:186} INFO - Started process (PID=76982) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T02:34:30.002-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-23T02:34:30.005-0500] {logging_mixin.py:190} INFO - [2025-02-23T02:34:30.005-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T02:34:30.015-0500] {logging_mixin.py:190} INFO - [2025-02-23T02:34:30.013-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-23T02:34:30.015-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T02:34:30.076-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.085 seconds
[2025-02-23T02:43:35.721-0500] {processor.py:186} INFO - Started process (PID=79120) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T02:43:35.722-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-23T02:43:35.725-0500] {logging_mixin.py:190} INFO - [2025-02-23T02:43:35.725-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T02:43:35.731-0500] {logging_mixin.py:190} INFO - [2025-02-23T02:43:35.730-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-23T02:43:35.731-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T02:43:35.765-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.052 seconds
[2025-02-23T02:44:38.066-0500] {processor.py:186} INFO - Started process (PID=79329) to work on /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T02:44:38.070-0500] {processor.py:914} INFO - Processing file /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py for tasks to queue
[2025-02-23T02:44:38.074-0500] {logging_mixin.py:190} INFO - [2025-02-23T02:44:38.073-0500] {dagbag.py:588} INFO - Filling up the DagBag from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T02:44:38.085-0500] {logging_mixin.py:190} INFO - [2025-02-23T02:44:38.080-0500] {dagbag.py:387} ERROR - Failed to import: /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
Traceback (most recent call last):
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py", line 252, in <module>
    def show_params(**kwargs) -> None:
  File "/home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/decorators/base.py", line 691, in decorator_factory
    return _TaskDecorator(
TypeError: _TaskDecorator() takes no arguments
[2025-02-23T02:44:38.087-0500] {processor.py:927} WARNING - No viable dags retrieved from /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py
[2025-02-23T02:44:38.140-0500] {processor.py:208} INFO - Processing /home/rafaelrb/anaconda3/lib/python3.9/site-packages/airflow/example_dags/example_params_ui_tutorial.py took 0.088 seconds
